packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -670,7 +670,7 @@ class SliverChildListDelegate extends SliverChildDelegate {
       return null;
     Widget child = children[index];
     final Key key = child.key != null? _SaltedValueKey(child.key) : null;
-    assert(child != null);
+    assert(child != null, 'child at index $index is null');
```

Comment:
Same here:
'The sliver's children must not contain null values, but a null value was found at index $index.'

Reply:
I was also thinking of adding an assertion after line 559 for the `children`s contents. WDYT?

Reply:
I think keeping just the assert where it is now is fine, since it will catch it regardless of which constructor was used. If you added it at line 559, you'd probably want to do add it at the other constructor as well (ie line 536).